developerportal/apps/home/migrations/0034_support_featuring_video_in_homepage.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 2.2.7 on 2019-11-10 21:23
+
+from django.db import migrations
+import wagtail.core.blocks
+import wagtail.core.fields
+import wagtail.images.blocks
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('home', '0033_rephrase_articles_as_posts'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='homepage',
+            name='featured',
+            field=wagtail.core.fields.StreamField([('post', wagtail.core.blocks.PageChooserBlock(page_type=['articles.Article', 'externalcontent.ExternalArticle'])), ('external_page', wagtail.core.blocks.StructBlock([('url', wagtail.core.blocks.URLBlock()), ('title', wagtail.core.blocks.CharBlock()), ('description', wagtail.core.blocks.TextBlock(required=False)), ('image', wagtail.images.blocks.ImageChooserBlock())])), ('video', wagtail.core.blocks.PageChooserBlock(page_type=['videos.Video']))], blank=True, help_text='Optional space for featured posts, videos or links, max. 4. Note that External Video is NOT allowed here.', null=True),
```

Comment:
This migration is a no-op one (in postgres terms) that just keeps the frozen model schema up to date. It doesn't alter any tables